<?xml version="1.0" encoding="utf-8"?>
<launch>

  <param name ="/use_sim_time" value="false"/>
  <node  pkg="cam_lidar_calib" name="cam_lidar_calib_nerian_left" type="cam_lidar_calib_node" output="screen">
  	<param name = "dx" type="double" value="0.0615"/>
  	<param name = "dy" type="double" value="0.0615"/>

  	<param name = "checkerboard_rows" type = "int" value = "9" />
  	<param name = "checkerboard_cols" type = "int" value = "6" />

  	<param name = "result_file" type="string" value="$(find calibration)/debug_data/draw/C_T_L_left_VLP.txt"/>
  	<param name = "result_rpy_file" type="string" value="$(find calibration)/debug_data/draw/rpy_txyz_left.txt"/>

  	<param name = "min_points_on_plane" type="int" value="850"/>
  	<param name = "num_views" type="int" value="15"/>

  	<param name = "camera_in_topic" type="string" value="/nerian_stereo/left_image"/>
  	<param name = "lidar_in_topic" type="string" value="/velodyne_points"/>

	<param name="x_min" value="0"/>
    <param name="x_max" value="6"/>
    <param name="y_min" value="-1.25"/>
    <param name="y_max" value="1.25"/>
    <param name="z_min" value="-0.5"/>
    <param name="z_max" value="2"/>
    <param name="ransac_threshold" value="0.01"/>

  	<param name = "cam_config_file_path" type="string" value="$(find cam_lidar_calib)/config/nerian_passthru_faccal_left.yaml" />
  </node>
</launch>
